core/service/src/client/mod.rs

```diff
@@ -690,6 +690,7 @@ impl Client {
                 None => {
                     tracing::warn!("Signature could not be verified for a CRS");
                     // do not insert
+                    continue;
```

Comment:
shouldn't we have a new test here? as far as i understand, there was a security problem, so we need a test to check it never comes back, CC @jot2re @dd23

Reply:
as far as I understand, it was an High issue, so we need tests. Even if it's a one-word fix (or, what prevents another person, in the future, to remove this word?). 

> non-production code
I don't know, but then why was it rated as High issue?

Reply:
We didn't have time to argue about rating and instead just fixed the issue. Indeed this part of the code is only used for internal tools and testing.

> what prevents another person, in the future, to remove this word?
That's not a good argument, as we simply cannot test every single line of code for its existence or removal.

Reply:
> That's not a good argument, as we simply cannot test every single line of code for its existence or removal.

(assuming it would be a real high-level issue) I really disagree with that. If there is a big security problem which is found, we need to assess that it will never be back. Especially since there was a scenario for a real-life problem (assuming again the scenario was correct).